recipe/meta.yaml

```diff
@@ -1,31 +1,19 @@
 {% set name = "pocl" %}
-{% set version = "1.5" %}
-{% set build_number = 11 %}
-{% set sha256 = "4fcf4618171727d165fc044d465a66e3119217bb4577a97374f94fcd8aed330e" %}
-{% set llvm_version = "10.0.0" %}
+{% set version = "1.6" %}
+{% set build_number = 0 %}
+{% set sha256 = "b0a4c0c056371b6f0db726b88fbb76bbee94948fb2abd4dbc8d958f7c42f766c" %}
+{% set llvm_version = "10.0.1" %}
+{% set osx_llvm_version = "11.0.0" %}
```

Comment:
```suggestion
{% if target_platform is defined and target_platform.startswith("osx-") %}
{% set llvm_version = "10.0.1" %}
{% else %}
{% set llvm_version = "11.0.0" %}
{% endif %}
```

Reply:
cd4451b

Thanks! I went looking for a variable like this, [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=jinja#templating-with-jinja) and [here](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#env-vars) and even [here](https://github.com/conda/conda-build/blob/1071bfe070c39dbfa5c12ded1942fd7e5326c7e0/conda_build/environ.py), but I couldn't find it. Do you think it would make sense to add that to the docs somewhere?

Reply:
(Or alternatively: If I missed it, where is it documented?)

Reply:
It's not documented yet. https://github.com/conda/conda-build/issues/4035